lua/lv-utils/init.lua

```diff
@@ -1,9 +1,9 @@
 local lv_utils = {}
 
 function lv_utils.reload_lv_config()
-  vim.cmd "source ~/.config/nvim/lv-config.lua"
-  vim.cmd "source ~/.config/nvim/lua/plugins.lua"
-  vim.cmd "source ~/.config/nvim/lua/lv-neoformat/init.lua"
+  vim.cmd("source " .. CONFIG_PATH .."/lv-config.lua")
```

Comment:
One last thing you need to do:
- Install [stylua](https://github.com/johnnymorganz/stylua)
- Possibly copy utils/.stylua.toml in the root director (~/.config/nvim) <- I don't know if necessary
- Save your code to reformat automatically

This time you can `rebase -i`, change pick to e (for edit), `git add -u` and then `git rebase --continue` + `git push -f`.
Alternatively you can also `git reset --soft HEAD^`, add and commit, then `git push -f`

Reply:
I applied stylua, everything look nice and clean.